common-requests/get-technical-support.md

```diff
@@ -1,8 +1,20 @@
 ---
 layout: page
 title: "Technical Support"
-description: "TBD"
+description: ""
+tags:
+ - Agent
+ - Help
+ - Support
+author: edeediong
 ---
 
-# Get Technical Support
+## Get Technical Support
 
+To get technical support on Open Horizon, join the LF Edge [Slack workspace](https://join.slack.com/t/lfedge/shared_invite/zt-7kavdtmq-SeyFzM2CEABBcKYGEVCgkw) if you haven't.
+Ensure you join the *#open-horizon-help* channel on LF Edge as you can post all technical and non-technical inquiries get a response from the help team.
```

Comment:
Chg to:

When you join the LF Edge *#open-horizon-help* slack channel, you can post technical and non-technical inquiries and receive help team responses.

Reply:
Done